instrumentation/pekko/pekko-http-1.0/javaagent/src/test/scala/io/opentelemetry/javaagent/instrumentation/pekkohttp/v1_0/PekkoHttpServerInstrumentationTestAsync.scala

```diff
@@ -32,4 +39,23 @@ class PekkoHttpServerInstrumentationTestAsync
     super.configure(options)
     options.setTestHttpPipelining(false)
   }
+
+  @Test
+  def successfulPostRequestWithParent(): Unit = {
```

Comment:
Test name indicates that you are testing something completely different than what your true goal is. Test should be name appropriately. Parts that you don't need should be removed, do you really need the `tracePARENT` stuff? Why not send a simple get? I'd guess that for what you want to test you don't need to make a http request at all, perhaps you could skip that altogether?

Reply:
I'll look into rewriting this test

Reply:
@laurit thanks for looking at the PR so quickly. I wrote a new test that does not use HTTP.